lib/src/presentation/bloc/favorites_bloc.dart

```diff
@@ -0,0 +1,78 @@
+import 'dart:async';
+
+import 'package:hearthstoneapp/src/core/util/api_response_state.dart';
+import 'package:hearthstoneapp/src/data/model/hearthstone_card.dart';
+import 'package:hearthstoneapp/src/data/repository/favorites_repository.dart';
+import 'package:hearthstoneapp/src/domain/bloc/i_favorites_bloc.dart';
+
+class FavoritesBloc extends IFavoritesBloc {
+  late FavoritesRepository favoritesRepository;
+
+  final StreamController<ApiResponse> _favoritesCardListController =
+      StreamController.broadcast();
+
+  FavoritesBloc({FavoritesRepository? repository}) {
+    favoritesRepository = repository ?? FavoritesRepository();
+  }
+
+  @override
+  Stream<ApiResponse> getCardListStream() {
+    return _favoritesCardListController.stream;
+  }
```

Comment:
you can replace this with an arrow function

Reply:
Done